index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+  <title>Ahmad Zamir Yousufi Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <img class="logo" alt="logo" src="images/zheaderlogo.png">
```

Comment:
Please wrap the logo with `<a>` tag so that in the future if you want to go to the home section you would easily do that.